tests/src/test/scala/cats/tests/ConstTests.scala

```diff
@@ -31,4 +32,10 @@ class ConstTests extends CatsSuite {
   checkAll("Const[Map[Int, Int], String]", OrderLaws[Const[Map[Int, Int], String]].eqv)
   checkAll("PartialOrder[Const[Set[Int], String]]", OrderLaws[Const[Set[Int], String]].partialOrder)
   checkAll("Order[Const[Int, String]]", OrderLaws[Const[Int, String]].order)
+
+  {
+    implicit val constContravariant = Const.constContravariant[Int]
```

Comment:
Was this necessary to get the law-checking to compile? It seems like we shouldn't have to do this.